src/Bases/Staker/TokenizedStaker.sol

```diff
@@ -33,7 +33,7 @@ abstract contract TokenizedStaker is BaseHooks, ReentrancyGuard {
          * @dev Used for lastRewardRate, a rewardRate equivalent for instant reward releases.
          */
         uint96 lastNotifyTime;
-        /// @notice The last rewardRate before a notifyRewardAmount was called
+        /// @notice The last rewardRate before a notifyRewardAmount was called scaled by PRECISION
```

Comment:
Not exactly correct...the rewardRate is not further scaled by precision; just important to know that this is not actually tokens per second, but tokens per second * `PRECISION`

Reply:
gonna go back to the original comment 

since now the definition of rewardRate implies that this